shopping-list/shopping.js

```diff
@@ -0,0 +1,59 @@
+import {     
+    logout,
+    createItem,
+    deleteItems,
+    purchasedItem,
+    checkAuth,
+    getItems } from '../fetch-utils.js';
+import { renderItem } from '../render.js';
+
+const form = document.querySelector('.shopping-form');
+const deleteButton = document.querySelector('.delete-button');
+const logoutButton = document.querySelector('.logout-button');
+const itemList = document.querySelector('.item-div-list');
+
+checkAuth();
+displayList();
```

Comment:
you have your load event listener on line 49 that does this for you so you can get rid of this function call on 16